pkg/controller/projectreference/projectreference_adapter.go

```diff
@@ -206,37 +206,37 @@ func getMatchingClaimLink(projectReference *gcpv1alpha1.ProjectReference, client
 }
 
 // updateProjectID updates the ProjectReference with a unique ID for the ProjectID
-func (r *ReferenceAdapter) updateProjectID() error {
+func (r *ReferenceAdapter) UpdateProjectID() error {
```

Comment:
Fix the comment `// UpdateProjectID`

speaking of it, it's not much trouble fix the comment also for https://github.com/openshift/gcp-project-operator/blob/master/pkg/controller/projectreference/projectreference_adapter.go#L276 which is referring to the wrong function.

Reply:
we don't need a comment for a private function, do we?